src/sqlfluff/core/rules/base.py

```diff
@@ -773,6 +773,10 @@ def discard_unsafe_fixes(
                 lint_result.fixes = []
                 return
 
+    @staticmethod
+    def _split_comma_separated_string(raw_str: str) -> List[str]:
```

Comment:
Nitpick: If we remove the `_` prefix, I think this function would show up in the "autodoc" docs for the `BaseRule` class, which could be helpful for other people working on rules.

Reply:
Will do. Also wonder if we should add BaseRule to https://docs.sqlfluff.com/en/stable/developingrules.html

Will open a separate PR for that.